lightningd/opening_control.c

```diff
@@ -278,6 +286,7 @@ static void funding_broadcast_success(struct channel *channel)
 	struct funding_channel *fc = channel->peer->uncommitted_channel->fc;
 	struct command *cmd = fc->cmd;
 
+	/* This shouldn't happen, but be explicit. */
```

Comment:
Why this https://github.com/ElementsProject/lightning/blob/ec485c816e553fa157e5489f1f0a67d629165777/lightningd/opening_control.c#L290-L295
should not happen ?

Reply:
Good catch!

Hmm, the commit which added that was correct (since cancel overrode success), but the next one made the comment incorrect.  Will remove comment altogether!